[UXE-2351] chore: migrate compliance to Docs (#840)
aziontech/docs: src/content/docs/pt-br/pages/menu-principal/compliance/soc.mdx

---
title: Compliance SOC
description: A Azion recebe relatórios anuais SOC 2 Tipo 2 e SOC 3, que demonstram que os controles de segurança são continuamente auditados ao longo do ano.
meta_tags: Azion, edge computing, compliance, segurança e privacidade de dados
namespace: documentation_compliance_soc

> permalink: /documentacao/produtos/compliance/soc/

Author: ```suggestion
permalink: /documentacao/compliance/soc/
```

---

## O que são o SOC 2 Tipo 2 e o SOC 3?

**SOC 2** é um procedimento de auditoria desenvolvido pelo **Instituto Americano de Contadores Públicos Certificados (American Institute of Certified Public Accountants - AICPA)** para garantir que os provedores de serviços tomem sempre as medidas necessárias para manter a segurança e a privacidade dos dados. O SOC 2 atesta a confiabilidade dos serviços prestados pela organização, sendo composto por cinco “princípios de serviço de confiança” que servem como base para o gerenciamento seguro de dados.

---

## A availação da Azion

A Azion é avaliada em dois princípios de confiança diferentes: **Segurança** e **Disponibilidade**.

Essa avaliação de terceiros aponta a capacidade da plataforma de impedir acesso não autorizado, violações e problemas relacionados à segurança que podem afetar a disponibilidade.

> O atestado **SOC 2 Tipo 2** da Azion se aplica a toda a Plataforma de Edge Computing da Azion, incluindo o Edge Application e todos os seus módulos (Application Accelerator, Edge Cache, Edge Functions e Load Balancer) e o Edge Firewall e seus módulos (DDoS Protection Unmetered, Network Layer Protection, Web Application Firewall - WAF e Origin Shield), cobrindo todas as regiões em que a rede da Azion está presente. A Azion se esforça para certificar toda a plataforma principalmente por um motivo: performance. Experiências perfeitas exigem que aplicações e dados sejam processados e entregues o mais próximo possível dos usuários finais, e se toda a rede e a plataforma não estiverem certificadas, isso levaria a diferenças de desempenho.

Author: ```suggestion
O atestado **SOC 2 Tipo 2** da Azion se aplica a toda a Plataforma de Edge Computing da Azion, incluindo o Edge Application e todos os seus add-ons (Application Accelerator, Edge Cache, Edge Functions e Load Balancer) e o Edge Firewall e seus add-ons (DDoS Protection Unmetered, Network Layer Protection, Web Application Firewall - WAF e Origin Shield), cobrindo todas as regiões em que a rede da Azion está presente. A Azion se esforça para certificar toda a plataforma principalmente por um motivo: performance. Experiências perfeitas exigem que aplicações e dados sejam processados e entregues o mais próximo possível dos usuários finais, e se toda a rede e a plataforma não estiverem certificadas, isso levaria a diferenças de desempenho.
```

> O atestado **SOC 2 Tipo 2** da Azion se aplica a toda a Plataforma de Edge Computing da Azion, incluindo o Edge Application e todos os seus módulos (Application Acceleration, Edge Cache, Edge Functions e Load Balancer) e o Edge Firewall e seus módulos (DDoS Protection Unmetered, Network Layer Protection, Web Application Firewall - WAF e Origin Shield), cobrindo todas as regiões em que a rede da Azion está presente. A Azion se esforça para certificar toda a plataforma principalmente por um motivo: performance. Experiências perfeitas exigem que aplicações e dados sejam processados e entregues o mais próximo possível dos usuários finais, e se toda a rede e a plataforma não estiverem certificadas, isso levaria a diferenças de desempenho.

Author: ```suggestion
O atestado **SOC 2 Tipo 2** da Azion se aplica a toda a Plataforma de Edge Computing da Azion, incluindo o Edge Application e todos os seus módulos (Application Accelerator, Edge Cache, Edge Functions e Load Balancer) e o Edge Firewall e seus módulos (DDoS Protection Unmetered, Network Layer Protection, Web Application Firewall - WAF e Origin Shield), cobrindo todas as regiões em que a rede da Azion está presente. A Azion se esforça para certificar toda a plataforma principalmente por um motivo: performance. Experiências perfeitas exigem que aplicações e dados sejam processados e entregues o mais próximo possível dos usuários finais, e se toda a rede e a plataforma não estiverem certificadas, isso levaria a diferenças de desempenho.
```

> O atestado **SOC 2 Tipo 2** da Azion se aplica a toda a Plataforma de Edge Computiimg da Azion, incluindo o Edge Application e todos os seus add-ons (Application Acceleration, Edge Cache, Edge Functions e Load Balancer) e o Edge Firewall e seus add-ons (DDoS Protection Unmetered, Network Layer Protection, Web Application Firewall - WAF e Origin Shield), cobrindo todas as regiões em que a rede da Azion está presente. A Azion se esforça para certificar toda a plataforma principalmente por um motivo: performance. Experiências perfeitas exigem que aplicações e dados sejam processados e entregues o mais próximo possível dos usuários finais, e se toda a rede e a plataforma não estiverem certificadas, isso levaria a diferenças de desempenho.

Reviewer: ```suggestion
O atestado **SOC 2 Tipo 2** da Azion se aplica a toda a Plataforma de Edge Computing da Azion, incluindo o Edge Application e todos os seus módulos (Application Acceleration, Edge Cache, Edge Functions e Load Balancer) e o Edge Firewall e seus módulos (DDoS Protection Unmetered, Network Layer Protection, Web Application Firewall - WAF e Origin Shield), cobrindo todas as regiões em que a rede da Azion está presente. A Azion se esforça para certificar toda a plataforma principalmente por um motivo: performance. Experiências perfeitas exigem que aplicações e dados sejam processados e entregues o mais próximo possível dos usuários finais, e se toda a rede e a plataforma não estiverem certificadas, isso levaria a diferenças de desempenho.
```

Mas isso não é tudo: a Azion deu um passo além, conquistando para os clientes o atestado **SOC 3**, com as mais rigorosas necessidades regulatórias e de compliance. O relatório SOC 3, atestado pelos mesmos auditores que verificaram os controles da Azion para o SOC 2 Tipo 2, traz informações relacionadas aos controles internos da Azion para segurança e disponibilidade, e pode ser obtido pelos clientes da Azion entrando em contato com seu **Customer Service Representative**.

---

## Como isso afeta os clientes da Azion?

As designações **SOC 2 Tipo 2** e **SOC 3** podem ser usadas para simplificar suas auditorias e garantir o nível de segurança dos serviços da plataforma de edge que você usa.

---

## FAQ

- **Quem realiza as auditorias independentes da Azion para seus relatórios SOC 2 Tipo 2 e SOC 3?**

Os relatórios SOC 2 Tipo 2 e SOC 3 da Azion, que abrangem os princípios do serviço de confiança de Segurança e Disponibilidade, são gerados por auditores externos terceirizados.

- **Com que frequência são emitidos os relatórios SOC 2 e SOC 3 da Azion?**

Os relatórios SOC 2 e SOC 3 da Azion são emitidos no mínimo uma vez por ano.

- **Quais regiões e produtos são cobertos?**

> Os relatórios SOC 2 e SOC 3 da Azion abrangem toda a plataforma de edge, o que inclui o Edge Application e todos os seus módulos (Application Acceleration, Edge Cache, Edge Functions e Load Balancer), o Edge Firewall e seus módulos (DDoS Protection Unmetered, Network Layer Protection, Web Application Firewall - WAF e Origin Shield), cobrindo todas as regiões em que a rede da Azion está implantada. Atualmente, a Azion está presente em seis continentes.

Author: ```suggestion
Os relatórios SOC 2 e SOC 3 da Azion abrangem toda a plataforma de edge, o que inclui o Edge Application e todos os seus módulos (Application Accelerator, Edge Cache, Edge Functions e Load Balancer), o Edge Firewall e seus módulos (DDoS Protection Unmetered, Network Layer Protection, Web Application Firewall - WAF e Origin Shield), cobrindo todas as regiões em que a rede da Azion está implantada. Atualmente, a Azion está presente em seis continentes.
```

> Os relatórios SOC 2 e SOC 3 da Azion abrangem toda a plataforma de edge, o que inclui o Edge Application e todos os seus add-ons (Application Acceleration, Edge Cache, Edge Functions e Load Balancer), o Edge Firewall e seus módulos (DDoS Protection Unmetered, Network Layer Protection, Web Application Firewall - WAF e Origin Shield), cobrindo todas as regiões em que a rede da Azion está implantada. Atualmente, a Azion está presente em seis continentes.

Reviewer: ```suggestion
Os relatórios SOC 2 e SOC 3 da Azion abrangem toda a plataforma de edge, o que inclui o Edge Application e todos os seus módulos (Application Acceleration, Edge Cache, Edge Functions e Load Balancer), o Edge Firewall e seus módulos (DDoS Protection Unmetered, Network Layer Protection, Web Application Firewall - WAF e Origin Shield), cobrindo todas as regiões em que a rede da Azion está implantada. Atualmente, a Azion está presente em seis continentes.
```

- **A Azion possui certificados de Compliance SOC 2 e SOC 3?**

> Não existe certificado de Compliance SOC 2. Em vez disso, avaliadores terceirizados qualificados produzem um relatório de compliance para a organização avaliada, atestando a confiabilidade dos serviços prestados. No caso do SOC 3, os clientes da Azion que contam com planos que incluem o Suporte de **Missão Crítica** podem solicitar uma cópia do relatório ao seu **Customer Service Representative**.

Reviewer 2: ```suggestion
Não existe certificado de Compliance SOC 2. Em vez disso, avaliadores terceirizados qualificados produzem um relatório de compliance para a organização avaliada, atestando a confiabilidade dos serviços prestados. No caso do SOC 3, os clientes da Azion que contam com planos que incluem o Suporte de **Mission Critical** podem solicitar uma cópia do relatório ao seu **Customer Service Representative**.
```

Planos não traduzimos :)
